src/components/visualizations/chart/events/load.ts

```diff
@@ -0,0 +1,24 @@
+// (C) 2007-2018 GoodData Corporation
+import { Chart } from 'highcharts';
+import get = require('lodash/get');
+
+// TODO: once Highcharts is upgraded to 7.0.1, 'yAxis.className' can be used to avoid registering 'load' event
+// We can add 'className' to 'yAxis' in 'chartOptionsBuilder.ts' and 'customConfiguration.ts'
+// (https://api.highcharts.com/highcharts/yAxis.className is supported since 5.0.0)
+/**
+ * Add class to Y axis to support selenium test for dual axes chart
+ * @param chart
+ */
+function addClassToYAxis(chart: Chart): void {
+    const yAxis: any[] = get(chart, 'yAxis', []);
```

Comment:
Please use `lodash/get` as little as possible: `const yAxis: any[] = chart && chart.yAxis || [];`

Reply:
Is there a type for `axis` in `highcharts`? If not, you can skip this one :-)

Reply:
Sorry, I can't remember.
But we should put it in the sync.
For example:
1. get(obj, 'a.b.c', {})
2. head(arr) or arr[0]
3. last(arr) or arr[arr.length - 1]